app/controllers/api/v1/documents.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'doorkeeper/grape/helpers'
+
+module API
+  module V1
+    class Documents < Grape::API
+      format :json
+
+      helpers Doorkeeper::Grape::Helpers
+
+      before do
+        doorkeeper_authorize!
+
+        def current_document
+          @current_document = Account.find(doorkeeper_token.resource_owner_id).document if doorkeeper_token
+        end
+      end
+
+      desc 'Documents related routes'
+      resource :document do
+        desc 'Return information about document'
+        get '/' do
+          current_document.as_json(only: %i[upload doc_type doc_number doc_expire])
+        end
+
+        desc 'Create new document'
+        params do
+          requires :profile_id, type: String, desc: 'Id of profile'
+          requires :upload,     type: String, desc: 'Document upload'
+          requires :doc_type,   type: String, desc: 'Type of document'
+          requires :doc_number, type: String, desc: 'Number of document'
+          requires :doc_expire, type: String, desc: 'Expiry date of document'
+        end
+        post '/create' do
+          Document.create!(profile_id: params[:profile_id],
+                           upload:     params[:upload],
```

Comment:
It would be grate to have tests or ability to test api through barong-client-app

Reply:
ok, I'm on it